Sources/StreamChatUI/CommonViews/SendButton/CooldownView.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+/// A view showing a cooldown when Slow Mode is active.
+open class CooldownView: _View, AppearanceProvider {
+    public struct Content {
+        public var cooldown: Int
+    }
+    
+    open var content: Content? {
+        didSet { updateContentIfNeeded() }
+    }
+    
+    public private(set) lazy var cooldownLabel: UILabel = UILabel()
+        .withoutAutoresizingMaskConstraints
+        .withAccessibilityIdentifier(identifier: "cooldownLabel")
+    
+    override open func updateContent() {
+        super.updateContent()
+        
+        guard let cooldown = content?.cooldown else { return }
+        cooldownLabel.text = "\(cooldown)"
```

Comment:
I would always assign the text. In the current shape, when the `nil` content is assigned, the label will be showing the previous text
```suggestion
        cooldownLabel.text = content.map { "\($0.cooldown)" }
```

Reply:
Changed